pages/getting-started.mdx

```diff
@@ -23,9 +24,19 @@ import CodeSnippet from '/components/code-snippet/CodeSnippet'
 
 Get Memgraph up and running on your data by following these installation steps. 
 
-<Steps>
 
-### Download and install Memgraph
+<Callout> 
+
+This installation script uses a `docker-compose.yml` file that we have prepared.
+You can download it from
+[https://github.com/memgraph/memgraph-platform](https://github.com/memgraph/memgraph-platform).
+Feel free to modify it according to your needs, and then run it using [Docker
```

Comment:
Maybe add link to **[configuration settings](https://memgraph.com/docs/configuration/configuration-settings)**